search_commands.go

```diff
@@ -293,8 +295,9 @@ type FTSearchSortBy struct {
 // More information about the options can be found
 // in the documentation for FT.SEARCH https://redis.io/docs/latest/commands/ft.search/
 type FTSearchOptions struct {
-	NoContent    bool
-	Verbatim     bool
+	NoContent bool
+	Verbatim  bool
+	// NoStopWords is deprecated since redis 8.0
```

Comment:
Maybe with this formatting the IDEs will pick it up as deprecated. Let's try.
```suggestion
	// Deprecated: NoStopWords is deprecated since Redis 8.0
```